src/modules/UI/scenes/Scan/Scan.ui.js

```diff
@@ -135,9 +136,9 @@ export default class Scan extends Component<Props> {
 
   renderCamera = () => {
     if (this.props.cameraPermission === AUTHORIZED) {
-      const torchMode = this.props.torchEnabled ? Camera.constants.TorchMode.on : Camera.constants.TorchMode.off
+      const torchMode = this.props.torchEnabled ? RNCamera.Constants.FlashMode.on : RNCamera.Constants.FlashMode.off
```

Comment:
why did this change from torch mode to flash mode?

Reply:
RNCamera replaced RCTCamera and so did the constants.

Reply:
are you sure you dont want `RNCamera.Constants.FlashMode.torch`

> RNCamera.Constants.FlashMode.torch turns on torch mode, meaning the flash light will be turned on all the time (even before taking photo) just like a flashlight.

https://github.com/react-native-community/react-native-camera/blob/master/docs/RNCamera.md#flashmode

Reply:
I think you are right. Can you submit a PR to change.